google/cloud/bigtable/examples/data_async_snippets.cc

```diff
@@ -22,6 +22,7 @@
 
 namespace {
 namespace cbt = google::cloud::bigtable;
+const std::string MAGIC_ROW_KEY = "key-000005";
```

Comment:
What's the agreed upon convention for constant naming here? Should this be `kMagicRowKey`?

Reply:
I used the same format as in data_snippets.cc
@coryan would know better.

Reply:
We have been terribly inconsistent with naming constants, we need to make a pass cleaning them up.  Unfortunately the clang-tidy checks break if you change some of these, so I have been leaving them alone until we figure out the correct incantations for clang-tidy.

Reply:
I did attempt it a while ago. Something like the following should work in theory:

```
- { key: readability-identifier-naming.GlobalConstantPrefix,    value: k }
- { key: readability-identifier-naming.GlobalConstantCase,     value: CamelCase }
```

If I recall, after I applied this pattern to the existing check options, it started complaining about about the include guards.